staging/src/k8s.io/cloud-provider/volume/helpers/rounding.go

```diff
@@ -117,6 +131,16 @@ func RoundUpToKiBInt(size resource.Quantity) (int, error) {
 	return roundUpSizeInt(requestBytes, KiB)
 }
 
+// RoundUpToGiBInt32 rounds up given quantity up to chunks of GiB. It returns an
+// int32 instead of an int64 and an error if there's overflow
+func RoundUpToGiBInt32(size resource.Quantity) (int32, error) {
```

Comment:
I don't think there's a test for this one, can we add one?

Reply:
Added.